WORKSPACE

```diff
@@ -100,6 +100,58 @@ load("@vizier_api_python_deps//:requirements.bzl", "install_deps")
 
 install_deps()
 
+# java_external
```

Comment:
I wasn't sure the best way to structure this. It appears most of the bazel code is refactored to into macros, etc. I tried to fit the rule installation into the existing pattern, but I wasn't sure the best way to layout this code.

Let me know if there is more appropriate way to structure this.